cmd/main.go

```diff
@@ -104,19 +119,31 @@ var cmd = &cobra.Command{
 			resp.Body.Close() // nolint
 
 			// Get the job URL from prow easy access
-			jobURL, buildID, err := getJobURL(jobInfo.ID)
+			jobStatus := JobStatus{JobID: jobInfo.ID}
+			err = wait.ExponentialBackoff(backoff, func() (done bool, err error) {
+				err2 := jobStatus.getJobURL()
+				if err2 != nil {
+					return false, err2
+				}
+				if len(jobStatus.JobURL) == 0 {
+					fmt.Println("Empty job URL will attempt retry")
+					return false, nil
+				}
+				return true, nil
+			})
+
 			if err != nil {
 				return err
 			}
 
 			jobRunIdentifier := JobRunIdentifier{
 				JobName:  opts.jobName,
-				JobRunID: buildID,
+				JobRunID: jobStatus.BuildID,
 			}
 			jobRunIdentifiers = append(jobRunIdentifiers, jobRunIdentifier)
 
 			// Print job info in tabular format
-			fmt.Printf(strFmt, strconv.Itoa(i+1), jobInfo.ID, jobURL)
+			fmt.Printf(strFmt, strconv.Itoa(i+1), jobInfo.ID, jobStatus.JobURL)
 
 			// Sleep to avoid hitting the api too hard
 			time.Sleep(time.Second)
```

Comment:
Can we kill the sleep? It's just a second but shouldn't be needed anymore

Reply:
That is a different sleep from the one in the loop getting the URL.  That one is in the loop for creating multiple jobs, if we remove it and we hit a case where we create a job and query in 5ms we will immediately create another job.  I think the intent was just to pause briefly so we don't hammer the api.  I don't see the downside to keeping it.

Reply:
Granted if we are only kicking off one job that sleep isn't really needed, suppose we could code around that if we wanted.